### PR TITLE
Feat (metrics): Video Color Densities

### PR DESCRIPTION
**Pull request by Gorkem-Encord**
Average Red, Green, and Blue colors for the videos. The aim is to have a playable video in the explorer (maybe after clicking the image in the grid view).

### CODE REVIEW

File: src/encord_active/db/models.py [hunk 1]
```diff
@@ -153,14 +153,17 @@ class ProjectDataAnalytics(SQLModel, table=True):
...
+    metric_custom0: Optional[float] = MetricFieldTypeNormal
     # Random
     metric_random: Optional[float] = MetricFieldTypeNormal
     # Image Only
     metric_object_count: Optional[int] = MetricFieldTypePositiveInteger
     metric_object_density: Optional[float] = MetricFieldTypeNormal
     metric_image_difficulty: Optional[float]  # FIXME: is the output of this always an integer??
     metric_image_singularity: Optional[float] = MetricFieldTypeNormal
-
+    metric_video_red_density: Optional[float] = MetricFieldTypeNormal
```
Comment (james-encord) [inline src/encord_active/db/models.py:164]: Could you add a comment to mark this as its own section of Data metrics.
# Whole video metrics or similar.

**Review by james-encord — CHANGES_REQUESTED** [review]

File: frontend_components/encord_active_components/frontend/src/components/explorer/Explorer.tsx [hunk 2]
```diff
@@ -1076,7 +1076,7 @@ const ImageWithPolygons = ({
           className="object-contain rounded transition-opacity"
           src={imgSrcUrl}
           muted
-          controls={false}
+          controls={true}
```
[the remark this answers is not in the document]
Comment (Gorkem-Encord) [reply frontend_components/encord_active_components/frontend/src/components/explorer/Explorer.tsx:1079]: Agree, this is a bit beyond my expertise so leaving it to you or @Encord-davids .

[the remark this answers is not in the document]
Comment (Encord-davids) [reply frontend_components/encord_active_components/frontend/src/components/explorer/Explorer.tsx:1075]: @frederik-encord Maybe something like "if video and average metric show controls"

**Comment by frederik-encord** [comment]
Closing this PR as relevant changes were merged into branch `feat/avg-simple-metrics`

**Pull request closed without merging**